automation/jenkins_build-containers.sh

```diff
@@ -19,11 +19,32 @@ fi
 
 for DOCKERFILE in "${DOCKERFILES[@]}"
 do
-  REPO_NAME=${DOCKERFILE#"Dockerfile_"}
+  REPO_NAME=${DOCKERFILE%".template"}
+  REPO_NAME=${REPO_NAME#"Dockerfile_"}
+  BUILDER="docker"
+  BUILDER_OPTS=""
+  REPO_NAME="${REPO_NAME}"
+  case ${DOCKERFILE} in
+    *template)
+      BUILDER="balena"
+      BUILDER_OPTS="--deviceType ${DEVICE_TYPE:-jetson-tx2} --arch ${DEVICE_ARCH:-aarch64}"
```

Comment:
References to tx2 and aarch64 need to be removed from these defaults

Reply:
Done.